aspnetcore/mvc/controllers/actions.md

```diff
@@ -14,80 +14,91 @@ uid: mvc/controllers/actions
+
+#### 3. A non-empty response body formatted in a content type negotiated with the client
+
+This type is better known as a **Content Negotiated Response**. [Content negotiation](xref:mvc/models/formatting#content-negotiation) applies whenever an action returns an [ObjectResult](https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.aspnetcore.mvc.objectresult) type or something other than an [IActionResult](https://docs.microsoft.com/en-us/aspnet/core/api/microsoft.aspnetcore.mvc.iactionresult) implementation. 
+
+Instead of returning an object directly, an action can return a content negotiated response. Some helper methods of this type include `Created`, `CreatedAtAction`, `CreatedAtRoute`, and `Ok`. Examples: `return Ok();` or `return CreatedAtRoute("routename", values, newobject);`
 
 ### Cross-Cutting Concerns
 
-In most apps, many actions will share parts of their workflow. For instance, most of an app might be available only to authenticated users, or might benefit from caching. When you want to perform some logic before or after an action method runs, you can use a *filter*. You can help keep your actions from growing too large by using [Filters](filters.md) to handle these cross-cutting concerns. This can help eliminate duplication within your actions, allowing them to follow the [Don't Repeat Yourself (DRY) principle](http://deviq.com/don-t-repeat-yourself/).
+In most apps, many actions share parts of their workflow. For instance, most of an app might be available only to authenticated users, or might benefit from caching. When you want to perform some logic before or after an action method runs, you can use a *filter*. You can help keep the actions from growing too large by using [Filters](xref:mvc/controllers/filters) to handle these cross-cutting concerns. This can help eliminate duplication within the actions, allowing them to follow the [Don't Repeat Yourself (DRY) principle](http://deviq.com/don-t-repeat-yourself/).
 
-In the case of authorization and authentication, you can apply the `Authorize` attribute to any actions that require it. Adding it to a controller will apply it to all actions within that controller. Adding this attribute will ensure the appropriate filter is applied to any request for this action. Some attributes can be applied at both controller and action levels to provide granular control over filter behavior. Learn more: [Filters](filters.md).
+In the case of authorization and authentication, you can apply the `Authorize` attribute to any actions that require it. Adding it to a controller applies it to all actions within that controller. Some attributes can be applied at both controller and action levels to provide granular control over filter behavior.
 
 Other examples of cross-cutting concerns in MVC apps may include:
```

Comment:
[Error handling](xref:mvc/controllers/filters#exception-filters) and  [Response Caching](xref:performance/caching/response) are often used as cross-cutting.  - Or something like that.